v2/oauth/multiple-frontends-with-separate-backends.mdx

```diff
@@ -0,0 +1,286 @@
+
+:::
+
+</TabItem>
+
+</BackendSDKTabs>
+
+
+###  Configure the Authorization Service Frontend 
+
+
+Initialize the `OAuth2Provider` recipe on the frontend of your **Authorization Service**. 
+
+:::info
+
+If you want to use your own custom UI check our [separate guide](/docs/oauth/customizations/custom-ui) that explains
+all the steps that you have to take into account.
+
+:::
+
+<OAuthFrontendTabs>
+
+
+<TabItem value="react">
+
+Update the `SuperTokensConfig` variable to include the `OAuth2Provider` recipe.
+
+1. Add the import statement for the recipe.
+
+```typescript 
+import OAuth2Provider from "supertokens-auth-react/recipe/oauth2provider";
+```
+
+2. Update the list of recipes to also include the new initialization.
+
+```tsx
+OAuth2Provider.init()
+```
+    
+</TabItem>
+
+
+<TabItem value="angular">
+
+Update the `AuthComponent` so that it also includes the `OAuth2Provider` recipe.
+You will have to add a new item in the `recipeList` array, inside the `supertokensUIInit` call.
+
+```typescript
+
+supertokensUIOAuth2Provider.init(),
+
+```
+
+</TabItem>
+
+<TabItem value="vue">
+
+Update the `AuthView` component so that it also includes the `OAuth2Provider` recipe.
+You will have to add a new item in the `recipeList` array, inside the `supertokensUIInit` call.
+
+```typescript
+
+supertokensUIOAuth2Provider.init(),
+
+```
+    
+</TabItem>
+    
+</OAuthFrontendTabs>
+
+
+
+## 4. Set Up Session Handling in Each Application
+
+In each of your individual `applications` you will have to setup up logic for handling the **OAuth 2.0** authentication flow.
+We recommend using a generic **OICD** or **OAuth** library in order to do this.
+
+<OAuthBackendTabs>
+
+<TabItem value="nodejs">
+
+You can use the [passport-oauth2](https://www.passportjs.org/packages/passport-oauth2/) library. 
+Just follow the instructions on the library's page and setup your application `backend`. 
+The configuration parameters can be determined based on the response that we received on **step 2**, when we created the **OAuth2 Client**.
+
+- `authorizationURL` corresponds to `authorize_url`
+- `tokenURL` corresponds to `token_fetch_url`
+- `clientID` corresponds to `client_id`
+- `clientSecret` corresponds to `client_secret`
+- `callbackURL` corresponds to a value from `callback_urls`
+- `scope` corresponds to `scope` 
+
+</TabItem>
+
+<TabItem value="go">
+
+You can use the [OAuth2](https://pkg.go.dev/golang.org/x/oauth2) library.
+Just follow these [instructions](https://golang.org/pkg/golang.org/x/oauth2/#example-Config-RequestToken) and implement it in your `backend`.
+
+</TabItem>
+
+
+<TabItem value="python"> 
+
+You can use the [AuthLib](https://docs.authlib.org/) library.
+Just follow these [instructions](https://docs.authlib.org/en/latest/client/oauth2.html) and implement it in your `backend`.
+
+</TabItem>
+
+
+</OAuthBackendTabs>
+
+
+## 5. Update the login flow in your frontend applications
```

Comment:
Does this step correctly apply to all the languages mentioned for the previous step?